docs/gamejam/cozy.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
+        <title>MakeCode Arcade Game Jam</title>
+        <meta name="twitter:card" content="summary_large_image">
+        <meta name="twitter:url" content="https://arcade.makecode.com/gamejam">
+        <meta name="twitter:title" content="MakeCode Arcade Game Jam">
+        <meta name="twitter:site" content="@MsMakeCode">
+        <meta name="twitter:creator" content="@MsMakeCode">
+        <meta name="twitter:description" content="The next MakeCode Arcade Game Jam starts on May 3rd! Get Ready!">
```

Comment:
```suggestion
        <meta name="twitter:description" content="Join the MakeCode Arcade Cozy Game Jam!">
```

Not sure if we event do tweets anymore, but just in case...